dbt/adapters/bigquery/impl.py

```diff
@@ -667,7 +689,7 @@ def load_dataframe(
         load_config.skip_leading_rows = 1
         load_config.schema = bq_schema
         load_config.field_delimiter = field_delimiter
-        with open(agate_table.original_abspath, "rb") as f:
+        with open(agate_table.original_abspath, "rb") as f:  # type: ignore
```

Comment:
I'm guessing the typecheck error is showing up since we are now specifying that `agate_table` is of type `agate.Table`. Do we know that this is true? Also, providing a new type annotation on only one argument feels odd. I would either do all (not preferred due to scope and intent of this PR), or do none (preferred).